man/systemd-nspawn.xml

```diff
@@ -336,6 +336,21 @@
       </varlistentry>
 
       <varlistentry>
+        <term><option>--pivot-root=</option></term>
+
+        <listitem><para>Pivot the specified directory to <filename>/</filename> inside the container, and either remove the
```

Comment:
uh, "remove" sounds like actually deleting files to me. Maybe just use "unmount" or so?